### DISCUSSION TITLE
deflate gizp response_body

### DISCUSSION THREAD

**Discussion by Git4Future** [apache/apisix#5913] [Q&A]
how to deflate gzip response body in apisix?
 is there any plugin or lua file,just like zlib in openresty
`local zip = require 'zlib'
local uncompress = zip.inflate()
local compress = zip.deflate()`

**Comment by juzhiyuan** [comment]
Hi @tzssangglass, please take a look 😄

**Reply by tzssangglass** [reply]
refer to: https://github.com/apache/apisix/blob/master/docs/en/latest/plugins/gzip.md
